app/localization/translated/zh.json

```diff
@@ -2194,10 +2194,10 @@
   "flakyTests.flakyTestsMatrixTooltipPossibleTimes": "可能的时间",
   "launchAnalysisModal.analyze": "分析",
   "launchAnalysisModal.baseOptions.allLaunches": "全部测试任务",
-  "launchAnalysisModal.baseOptions.current": "仅限当前测试任务",
-  "launchAnalysisModal.baseOptions.currentAndWithSameName": "当前的启动和具有相同名称的启动",
-  "launchAnalysisModal.baseOptions.previousLaunch": "之前具有相同名称的启动",
-  "launchAnalysisModal.baseOptions.withSameName": "有相同名称的测试任务",
+  "launchAnalysisModal.baseOptions.current": "目前推出",
+  "launchAnalysisModal.baseOptions.currentAndWithSameName": "所有启动项均具有相同名称",
+  "launchAnalysisModal.baseOptions.previousLaunch": "仅上次启动时同名",
+  "launchAnalysisModal.baseOptions.withSameName": "之前发布的同名作品",
```

Comment:
Please don't translate Chinese via translator tools. Leave the English version if the particular message is updated.